src/main/scala/io/iohk/ethereum/blockchain/sync/FastSyncNodesRequestHandler.scala

```diff
@@ -76,11 +78,14 @@ class FastSyncNodesRequestHandler(
   }
 
   private def handleMptNode(mptNode: MptNode): Seq[HashType] = mptNode match {
-    case n: MptLeaf =>
-      val evm = n.getAccount.codeHash
-      val storage = n.getAccount.storageRoot
+    case n: LeafNode =>
+      import AccountImplicits._
+      val account = n.value.toArray[Byte].toAccount
```

Comment:
Should we maybe handle the case that this conversion were to fail? As the node was received over the network it could be the case that it's invalid and doesn't contain an account, however it's up to discussion.